packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayerContainer.js

```diff
@@ -24,6 +28,21 @@ class InteractionLayerContainer extends Component {
     this.onPointerUp = this.onPointerUp.bind(this)
   }
 
+  componentDidMount () {
+    this.storeSVG()
+  }
+
+  componentDidUpdate (prevProps) {
+    if (this.props.svg !== prevProps.svg) {
+      this.storeSVG()
+    }
+  }
+
+  storeSVG () {
```

Comment:
Can we rename the class method `svgStore` so that it is not identical to the store method? At first, I thought the class method was calling itself.  :)